src/reactviews/pages/QueryResult/table/utils.ts

```diff
@@ -75,3 +75,18 @@ export function tryCombineSelections(selections: ISlickRange[]): ISlickRange[] {
     }
     return [unifiedSelection];
 }
+
+export function deepClone<T>(obj: T): T {
```

Comment:
Looks like this is [from VSCode](https://github.com/microsoft/vscode/blob/5bd3d12fb18047ae33ac4b171dee3cd044b6f3d4/src/vs/base/common/objects.ts#L8)?  Can you add a comment with a link to the source in case we need to update it.